spec/results-view-spec.coffee

```diff
@@ -319,15 +319,17 @@ describe 'ResultsView', ->
         itemHeight = resultsView.find('.selected').outerHeight()
         pageHeight = Math.round(resultsView.innerHeight() / itemHeight) * itemHeight
         initialScrollTop = resultsView.scrollTop()
+        itemsPerPage = Math.floor(pageHeight / itemHeight)
+        originalItemIndex = Math.floor(initialScrollTop / itemHeight) + itemsPerPage
```

Comment:
Ugh, my non-native language skills strike again. The idea with this variable is to use it as a starting point for the next specs, so that we can write stuff like `originalItemIndex - itemsPerPage` after triggering a page up event.

We're already asserting that such index is selected initially a couple of lines above, so I'd say the confusion probably comes from the name of the variable. What about `selectedIndexBeforePageUp`?